docker/nginx/conf.d/client.conf

```diff
@@ -25,6 +25,10 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
+	# parse subdomain into custom variable
+	server_name ~^(.*)?\.siasky\.dev|siasky\.dev$;
+	set $subdomain $1;
```

Comment:
What happens if I'll try to access [https://germany.siasky.net](https://germany.siasky.net) ? Will it assume `germany` is the skylink?

Reply:
We can fix that by making the regex match exactly N characters.

Reply:
Is it possible that the `server_name` wouldn't contain the `siasky` domain reference? Maybe some wildcard as a domain name would do the trick? I did my best not to hardcode our domain name (unless it was completely necessary) and instead used environment variables.

Reply:
https://regex101.com/r/If5280/1

Reply:
@kwypchlo we could also make it `^([a-z0-9]{55}).*?$` wdyt? would that work for you?